iml-manager-env/src/lib.rs

```diff
@@ -218,6 +219,40 @@ pub fn get_use_stratagem() -> bool {
     string_to_bool(get_var("USE_STRATAGEM"))
 }
 
+pub fn get_sfa_endpoints() -> Option<Vec<Vec<Url>>> {
+    let num = env::var("SFA_ENDPOINTS");
+
+    let num = match num {
+        Err(env::VarError::NotPresent) => None,
+        Err(e) => panic!("{}", e),
+        Ok(num) => Some(num),
+    }?;
+
+    let num = num
+        .parse::<u32>()
+        .unwrap_or_else(|_| panic!("Could not parse SFA_ENDPOINTS={} to a u32", num));
+
+    let xs = (1..=num).fold(vec![], |mut acc, x| {
+        let xs: Vec<_> = get_var(&format!("SFA_ENDPOINTS{}", x))
```

Comment:
I wonder if we could parse any env vars starting with `SFA_ENDPOINT`, for example: `SFA_ENDPOINT1`, `SFA_ENDPOINT_foo`, etc.

Reply:
I'm not sure what you mean, this will only parse endpoints in the range from 1 to num.

Reply:
We have to set at least two variables `SFA_ENDPOINTS` and `SFA_ENDPOINTSXXX`, right?

I am talking about a less error-prone design: keep only `SFA_ENDPOINTXXX`, just find them all by prefix. As a side effect we could have any format for `SFA_ENDPOINTXXX`: `SFA_ENDPOINTS_foo`, `SFA_ENDPOINTS_f1_qwerty`, etc.

Reply:
Ah. 

I suppose we could iterate all the environment variables and pick out the ones that match

`SFA_ENDPOINTS<NUM>`, however I don't know if that's less error prone. 

We're trying to simulate an array with environment variables here which has a length and values for each index.

Reply:
It would still be an array, we could even sort it :) It would make trivial to remove some URLs, etc.

Reply:
Sounds reasonable. I'll take a look at what's needed to change.